rfd/0107-abandon-active-access-request.md

```diff
@@ -0,0 +1,103 @@
+---
+authors: Jeff Pihach (jeff.pihach@goteleport.com)
+state: draft
+---
+
+# RFD 0107 - Abandon Active Access Requests
+
+## Required Approvers
+
+- Engineering: @zmb3
+
+## What
+
+Allow users who are using elevated permissions via an access request to abandon
+them ahead of their expiry once they are no longer of use.
+
+## Why
+
+By reducing the amount of time that a user has elevated permissions we also
+reduce the time and surface area to which an attack could occur in the event
+the users device becomes compromised.
+
+## Success Criteria
+
+- A user is able to abandon an active access request ahead of their expiry time
+  via the CLI or UI.
+- Creating a lock to prevent further use of the existing certificate.
+- An abandoned access request is unrecoverable and unusable.
+
+## Details
+
+### CLI
+
+A new `abandon` sub command to `tsh request` will be added. This command will
```

Comment:
Let's see if there are any other thoughts as to what to call this.

"Abandon" gives hint of "stale" or "outdated" and this feature is actually the opposite, it's allowing us to finalize a request before it expires.

Maybe "complete" or "finalize" are better verbs.